option画面関連ファイルの追加

KK358 opened the pull request:
## 概要

- option画面の作成

- DBのユーザーテーブルの変更によるエンティティクラス及びDBクラスの修正

- ランキング画面の作成

KK358 committed:
option画面関連ファイルの追加

FujimotoKensuke commented:
- オプション画面にユーザー名の入力フィールド、及び、処理の追加
- DeleteFlg.ON / DisplayFlg.ON のようなEnumを作成する
- インデントはnetbeansのデフォルトが8文字だった気がするのでオプションで設定

FujimotoKensuke commented on texasHoldem/src/main/java/db/UserTableDb.java:
```diff
@@ -24,6 +24,16 @@
+         */
+        public void updateComeents(String userId, String comments){
+            em.createQuery("UPDATE Usertable u SET u.comments = '" + comments + "' WHERE u.userId = '" + userId  + "'").executeUpdate();
+            
```
こちらの書き方に統一
Usertable user = em.createQuery("select u from Usertable u"
 				+ " where u.userId = :userId"
 				+ " AND u.password = :password", Usertable.class)
 				.setParameter("userId", ub.getUserId())
 				.setParameter("password", ub.getPassword())
 				.getSingleResult();

KK358 committed:
TexasHoldemEnum.javaにdeleteFlgとdisplyFlgを追加
userTableDb.javaのcreateQueryメソッドの中の書き方を統一
Usertable.javaのnullableの許容を変更
option画面にユーザー名の変更を追加
